test/client/fixtures/sandbox/node/dom-processor-test.js

```diff
@@ -432,3 +432,27 @@ test('Node.replaceChild must be overridden (GH-264)', function () {
     style.replaceChild(styleTextNode2, styleTextNode1);
     ok(style.innerHTML.indexOf(urlUtils.getProxyUrl('/image2.png')) > -1);
 });
+
+asyncTest('Script error on adding new element to non-added to DOM "body" element (GH-296)', function () {
```

Comment:
Or should we rather begin test names with `No error occurs when...` if this is what we expect?